### PR TITLE
HWKAPM-655 | python example app with kafka transport

### PR DESCRIPTION
**Pull request by pavolloffay**
https://issues.jboss.org/browse/HWKAPM-655

Python example application with kafka transport. Go application will be added in next PR.

@objectiser could you please review?

### CODE REVIEW

**Comment by pavolloffay** [comment]
Modified images(ruby, swarm, python) have been pushed to the hub.

File: examples/polyglot-zipkin/python-pyramid/README.adoc [hunk 1]
```diff
@@ -0,0 +1,16 @@
+= Example application of Python with pyramid_zipkin and swagger_zipkin
+
+Python application instrumented with pyramid_zipkin and swagger_zipkin.
+
+== Standalone Run & Build
+[source,shell]
+----
+$ go install github.com/pavolloffay/hello
```
Comment (objectiser) [inline examples/polyglot-zipkin/python-pyramid/README.adoc]: Not sure we should point to your repo - is there any other way to run this standalone from the example in apm?

Comment (pavolloffay) [inline examples/polyglot-zipkin/python-pyramid/README.adoc]: :smiley: oh I edited a wrong readme yesterday.

File: examples/polyglot-zipkin/README.adoc [hunk 2]
```diff
@@ -13,10 +13,11 @@ services.
 == Run
 [source,shell]
 ----
--> run tracing server (Hawkular APM or zipkin server). If using Hawkular APM use /standalone.sh -b 0.0.0.0
+-> run tracing server (Hawkular APM or zipkin server). If using Hawkular APM use /standalone.sh -b 0.0.0.0.
+-> Zipkin server has to be started with KAFKA_ZOOKEEPER=172.17.0.1:2181 java -jar zipkin-server.jar
```
Comment (objectiser) [inline examples/polyglot-zipkin/README.adoc]: should this be zipkin.jar instead of zipkin-server.jar?

**Pull request merged**